packages/mobile/src/identity/verification.ts

```diff
@@ -53,7 +53,7 @@ export const NUM_ATTESTATIONS_REQUIRED = 3
 export const VERIFICATION_TIMEOUT = 5 * 60 * 1000 // 5 minutes
 export const ERROR_DURATION = 5000 // 5 seconds
 export const NULL_ADDRESS = '0x0000000000000000000000000000000000000000'
-
+export const REQUEST_TX_GAS = 5708274
```

Comment:
Where did this magic # come from? What happens if it's too low?

Reply:
Since estimating gas does not work for concurrent txs, the only way to do this is to define it statically. That number I added just happened to be the gas estimation that I received for my request on Alfajores, but I should probably pad it by 2-3x. Will add it, but also happy to hear alternatives since it's certainly not elegant